recipes/hkl/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+
+test -d m4 || mkdir m4
+# gtkdocize || exit 1
+
+export ACLOCAL_PATH="$PREFIX/share/aclocal"
+aclocal --print-ac-dir
+
+autoreconf -ivf
```

Comment:
```suggestion
chmod +x autogen.sh
./autogen.sh
```

Reply:
I wanted to use it originally, but https://github.com/picca/hkl/blob/e3215664bebb2c37020499c6636ebed4f65566c7/autogen.sh#L4 specifically requires `gtkdocize`, which is supplied via the `gtk-doc` package, which does not exist for conda yet. Also, `ACLOCAL_PATH` is not set in that script, so I'd rather keep the existing changes. Is it a compeling argument?